cmd/security-agent/subcommands/runtime/activity_dump.go

```diff
@@ -134,7 +134,7 @@ func generateDumpCommands(globalParams *command.GlobalParams) []*cobra.Command {
 	activityDumpGenerateDumpCmd := &cobra.Command{
 		Use:   "dump",
 		Short: "generate an activity dump",
-		RunE: func(cmd *cobra.Command, args []string) error {
+		RunE: func(_ *cobra.Command, _ []string) error {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Would it still match the new rule if we used `_` as a prefix, meaning `_cmd` and `_args` for unused params?